impl/src/main/java/org/apache/myfaces/resource/FacesJSResourceLoader.java

```diff
@@ -95,7 +96,7 @@ public InputStream getResourceInputStream(ResourceMeta resourceMeta)
                 getHttpServletRequest(FacesContext.getCurrentInstance().getExternalContext());
 
         //name mapping does not happen on meta level
-        if(!(libraryName.equals(JsfLibrary.NAMESPACE) &&
+        if(!Objects.equals(libraryName,null) && !(libraryName.equals(JsfLibrary.NAMESPACE) &&
```

Comment:
```suggestion
        if(!JsfLibrary.NAMESPACE.equals(libraryName)) &&
```

Joshua Bloch always recommends flipping to use the Constant first this way you can't have an NPE